src/Viewer.tsx

```diff
@@ -458,11 +461,14 @@ function Viewer(): ReactElement {
             datasetKey = datasetParam || newCollection.getDefaultDatasetKey();
           } catch (error) {
             console.error(error);
-            notificationApi["error"]({
-              message: "Error loading dataset:",
-              description: (error as Error).message,
-              placement: "bottomLeft",
-              duration: 4,
```

Comment:
This error message flashes pretty quickly, so I decided to replace it with the new alert banner. It includes more helpful context and an immediate link for users to return to the homepage!